compiler/p/codegen/OMRTreeEvaluator.cpp

```diff
@@ -5968,48 +6051,52 @@ TR::Register *OMR::Power::TreeEvaluator::setmemoryEvaluator(TR::Node *node, TR::
    generateConditionalBranchInstruction(cg, TR::InstOpCode::bdnz, node, loopStartLabel, cndReg);
 
    generateLabelInstruction(cg, TR::InstOpCode::label, node, residualLabel); //check 16 aligned
```

Comment:
major performance benefit:  on Power10 (and future), the residue store can use vector-store with length.  without test-and-branch and only need a few instructions.  see inlined arrayCopy for example.